config/config.go

```diff
@@ -68,6 +68,99 @@ func NewCloudConfig(contents string) (*CloudConfig, error) {
 	return &cfg, err
 }
 
+func (co *CoreOS) Merge(mergeWith CoreOS) {
```

Comment:
Hey -- I'll be taking over dev here. Will issue a new pull request with your changes in a bit, once I understand these issues. Any suggestions on where to place it then? Place it all in initialize package so it gets handled at initialize.Apply time? In that case maybe we can get away with no merging and just applying each cloud config chunk serially. Will give it a shot.